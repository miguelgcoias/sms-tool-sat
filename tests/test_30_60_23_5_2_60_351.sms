30
47 19 85 1 19
37 6 49 4 1 1 2 2
24 6 36 5 1 2 1 1 1
1 20 21 1 20
22 6 34 3 2 2 2
7 1 8 1 1
37 9 46 3 6 1 2
16 12 40 5 3 3 2 3 1
24 18 42 2 5 13
10 20 50 3 6 9 5
43 19 81 3 5 10 4
36 15 51 5 3 3 3 3 3
0 15 15 3 5 7 3
9 8 17 1 8
35 10 55 2 4 6
6 21 27 5 5 4 2 4 6
28 6 40 4 2 2 1 1
56 11 78 4 2 5 1 3
34 4 42 2 2 2
28 22 72 5 8 3 2 5 4
34 11 56 1 11
37 5 42 2 1 4
29 6 35 5 1 1 1 1 2
7 6 13 2 2 4
8 9 17 5 1 1 2 2 3
12 22 56 3 8 3 11
30 18 48 1 18
17 16 33 2 6 10
25 17 59 2 7 10
52 19 71 5 4 5 3 6 1
3 7 24 14
3 26 14 28
2 13 4
0
1 6
0
3 13 28 5
1 24
1 4
4 8 5 14 16
13 19 28 6 23 10 4 5 26 14 24 7 9 12
4 17 24 26 25
0
1 6
3 29 9 2
0
3 6 24 14
8 15 25 26 22 24 17 2 12
5 13 25 4 14 28
8 10 27 15 8 26 23 25 7
7 6 14 28 4 2 16 10
2 26 10
2 24 14
0
0
2 14 13
2 16 6
1 24
5 5 14 28 17 8
6 27 15 28 2 24 20
