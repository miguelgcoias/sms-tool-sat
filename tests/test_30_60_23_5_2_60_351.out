3
5 22 24 26
6 7
14 9
